### ISSUE TITLE
[DOC] eli5 top-level API docs are incomplete

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#212]
Just noticed that http://eli5.readthedocs.io/en/latest/autodocs/eli5.html contains only explain_* and show_* functions. It does not have various formatters functions, and library-specific function. Options:

1. leave as is
2. add formatter functions (format_as_dict, etc.)
3. add everything in the top-level namespace (including library-specific functions)

I think currently the most important functions are described so it's fine to leave it as-is. But maybe we can also add formatter functions to increase their discoverability?

**Comment by kmike** [comment]
I think that'd be good to document formatter functions as top-level. I'd not document library-specific functions, because they should be rarely used by users, and we need to import them at top level more because of the way singledispatch discovery works, not because there is a real need to do it.
